common/changes/office-ui-fabric-react/u-mahuangh-fix-selected-item-keying_2019-03-11-16-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Key wrapped selectedItem children when they are keyed",
+      "type": "patch"
```

Comment:
FYI @Adjective-Object, if we modify the API file by introducing a backwards compatible or new method / member the change type should be `minor`. 

The type `patch` is reserved for bug fixes and the like which do not impact our public API.

I'll make the change here.